SPEC.md

```diff
@@ -128,15 +131,29 @@ Examples: `1.2.3`, `>= 1.0.0` or `~> 2.0`.
 
 A local path (String).
 
-The library will be linked from the local path. The `version` attribute
-isn't required but will be used if present to validate the dependency.
+The library will be linked from the local path. The `version` attribute is not
+required but will be used if present to validate the dependency.
 
 #### git
 
 A Git repository URL (String).
 
+The URL may be of [any protocol supported by the git command line client](https://git-scm.com/docs/git-clone#_git_urls_a_id_urls_a),
+which may include:
+
+- `ssh://`
+- `git://`
+- `https://`
+- `http://`
+- `ftps://`
+- `ftp://`
+- implied ssh protocol (e.g., `user@example.com:subdir/repo.git`)
+
+Just the same as the local git command line client, `ssh_config` and associated
+`netrc` settings will be honored for authentication-protected repositories.
```

Comment:
We do rely on the `Git` CLI but we may eventually choose to use `libgit2` or another solution which may not support these. I'm not sure we should include these in the SPEC.